### DISCUSSION TITLE
EE7 compatible?

### DISCUSSION THREAD

**Discussion by robinsowell** [Patrick64/reefine#12] [General]
Really neat add-on!  I've had someone ask me about EE7 compatibility, and looking at it, seems like it should be fine.  Any known issues?  On the store, it still just lists compatibility through EE6, but looks like it should be 7 compliant as well.

**Comment by Patrick64** [comment]
Thanks. I haven't tested it on EE7 but no one has come back saying it doesn't work so should be ok.

**Comment by robinsowell** [comment]
I took a quick look as well and nothing jumped out.  I'll tell them to go for it.

Thanks for the add-on, it really saves work on the frontend!
